store/storetest/channel_store.go

```diff
@@ -1115,6 +1116,70 @@ func testChannelStoreGetChannels(t *testing.T, ss store.Store) {
 	ss.Channel().InvalidateAllChannelMembersForUser(m1.UserId)
 }
 
+func testChannelStoreGetChannelsOpt(t *testing.T, ss store.Store) {
```

Comment:
Can we add tests with `ChannelTypes` filter and also a combined filter `ChannelTypes` and `UserIds`

Reply:
added